lib/torque/postgresql/associations/belongs_to_many_association.rb

```diff
@@ -179,7 +185,8 @@ def read_records_ids(records)
           def ids_rewriter(ids, operator)
             list = owner[source_attr] ||= []
             list = list.public_send(operator, ids)
-            owner[source_attr] = list.uniq.compact
+            list = list.uniq.compact.presence
```

Comment:
@crashtech what do you think about this change?

I did it because I have code which:

- In callback does `project.employees = [employee]`
- and somewhere else, does `project.employees << employee` for the same employee

I noticed in the DB it's persisted well (only one value in the array), but in my callbacks, I have an array with duplicate values, and this happens because `update_attribute` is called without uniq.